awx/main/tests/unit/settings/test_k8s_resource_setttings.py

```diff
@@ -57,5 +60,8 @@ def test_SYSTEM_TASK_ABS_CPU_conversion(value, converted_value, cpu_capacity):
         mock_settings.SYSTEM_TASK_ABS_CPU = value
         mock_settings.SYSTEM_TASK_FORKS_CPU = 4
         assert convert_cpu_str_to_decimal_cpu(value) == converted_value
-        assert get_corrected_cpu(-1) == converted_value
+        assert get_corrected_cpu(8) == converted_value
         assert get_cpu_effective_capacity(-1) == cpu_capacity
+        # SYSTEM_TASK_ABS_CPU should not effect cpu count and capacity for execution nodes
+        assert get_corrected_cpu(2.0, 'execution') == 2.0
+        assert get_cpu_effective_capacity(2.0, 'execution') == 8
```

Comment:
get_cpu_effective_capacity now takes a boolean as the second argument, so you can pass `is_control_node=False` to test the execution node scenario